background/openusercss-api.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+// begin:nanographql - Tiny graphQL client library
+// License: MIT
+
+const getOpname = /(query|mutation) ?([\w\d-_]+)? ?\(.*?\)? \{/;
+const gql = str => {
+  str = Array.isArray(str) ? str.join('') : str;
+  const name = getOpname.exec(str);
+  return function (variables) {
+    const data = {query: str};
+    if (variables) data.variables = JSON.stringify(variables);
+    if (name && name.length) {
+      const operationName = name[2];
+      if (operationName) data.operationName = name[2];
+    }
+    return JSON.stringify(data);
+  };
+};
+
+// end:nanographql
+
+const api = 'https://api.openusercss.org';
+const doQuery = ({id}, queryString) => new Promise((resolve, reject) => {
```

Comment:
No need for a new Promise, you can chain on `fetch` and return it. To reject simply return `Promise.reject(whatever)`